xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-rss/src/main/java/org/xwiki/rendering/internal/macro/rss/DefaultRomeFeedFactory.java

```diff
@@ -43,6 +44,9 @@
      * The maximum number of milliseconds to wait when inquiring the RSS feed provider.
      */
     private static final int TIMEOUT_MILLISECONDS = 5000;
+    private static final String USER_AGENT_HEADER = "User-Agent";
+    private String version = this.getClass().getPackage().getImplementationVersion();
```

Comment:
I feel those fields would make more sens as static, for example:

    private static final String VERSION = DefaultRomeFeedFactory.class.getPackage().getImplementationVersion();